examples/sphere/shallow_water.jl

```diff
@@ -256,6 +281,69 @@ elseif test_name == steady_state_compact_test_name
             local_geometry,
         )
 
+        return (h = h, u = u)
+    end
+elseif test_name == barotropic_instability_test_name
+    y0 = map(Fields.local_geometry_field(space)) do local_geometry
+        coord = local_geometry.coordinates
+
+        ϕ = coord.lat
+        λ = coord.long
+
+        if α == 0.0
+            ϕprime = ϕ
+            λprime = λ
+        else
+
+            ϕprime = asind(sind(ϕ) * cosd(α) - cosd(ϕ) * cosd(λ) * sind(α))
+            λprime = asind(sind(λ) * cosd(ϕ) / cosd(ϕprime)) # for alpha45, this experiences numerical precision issues. The test case is designed for either alpha0 or alpha30
```

Comment:
It seems `λprime` is not used in the initial conditions. Maybe just remove it?

Reply:
Yes, thanks for the good eye!